apps/examples/rt_ocf_test/rt_ocf_test_main.c

```diff
@@ -0,0 +1,738 @@
+	0x32, 0x2d, 0x32, 0x32, 0x32, 0x32, 0x2d, 0x32, 0x32, 0x32, 0x32, 0x2d,
+	0x32, 0x32, 0x32, 0x32, 0x2d, 0x32, 0x32, 0x32, 0x32, 0x32, 0x32, 0x32,
+	0x32, 0x32, 0x32, 0x32, 0x32, 0x69, 0x72, 0x65, 0x73, 0x6f, 0x75, 0x72,
+	0x63, 0x65, 0x73, 0x81, 0xa1, 0x64, 0x68, 0x72, 0x65, 0x66, 0x6b, 0x2f,
+	0x6c, 0x69, 0x67, 0x68, 0x74, 0x2f, 0x62, 0x6c, 0x75, 0x65, 0x6a, 0x70,
+	0x65, 0x72, 0x6d, 0x69, 0x73, 0x73, 0x69, 0x6f, 0x6e, 0x1b, 0x00, 0x00,
+	0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x6a, 0x72, 0x6f, 0x77, 0x6e, 0x65,
+	0x72, 0x75, 0x75, 0x69, 0x64, 0x78, 0x24, 0x34, 0x34, 0x34, 0x34, 0x34,
+	0x34, 0x34, 0x34, 0x2d, 0x34, 0x34, 0x34, 0x34, 0x2d, 0x34, 0x34, 0x34,
+	0x34, 0x2d, 0x34, 0x34, 0x34, 0x34, 0x2d, 0x34, 0x34, 0x34, 0x34, 0x34,
+	0x34, 0x34, 0x34, 0x34, 0x34, 0x34, 0x34
+};
+
+static const char TEMP_PSTAT_DATA[] = {
+	0xa7, 0x63, 0x64, 0x6f, 0x73, 0xa2, 0x61, 0x73, 0x1b, 0x00, 0x00, 0x00,
+	0x00, 0x00, 0x00, 0x00, 0x03, 0x61, 0x70, 0xf4, 0x64, 0x69, 0x73, 0x6f,
+	0x70, 0xf5, 0x62, 0x63, 0x6d, 0x1b, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+	0x00, 0x00, 0x62, 0x74, 0x6d, 0x1b, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+	0x00, 0x00, 0x62, 0x6f, 0x6d, 0x1b, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+	0x00, 0x04, 0x62, 0x73, 0x6d, 0x1b, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+	0x00, 0x04, 0x6a, 0x72, 0x6f, 0x77, 0x6e, 0x65, 0x72, 0x75, 0x75, 0x69,
+	0x64, 0x78, 0x24, 0x34, 0x34, 0x34, 0x34, 0x34, 0x34, 0x34, 0x34, 0x2d,
+	0x34, 0x34, 0x34, 0x34, 0x2d, 0x34, 0x34, 0x34, 0x34, 0x2d, 0x34, 0x34,
+	0x34, 0x34, 0x2d, 0x34, 0x34, 0x34, 0x34, 0x34, 0x34, 0x34, 0x34, 0x34,
+	0x34, 0x34, 0x34
+};
+
+static FILE *test_doxm_fopen(const char *path, const char *mode)
+{
+	(void)path;
+	return fopen(TEMP_DOXM_PATH, mode);
+}
+
+static FILE *test_cred_fopen(const char *path, const char *mode)
+{
+	(void)path;
+	return fopen(TEMP_CRED_PATH, mode);
+}
+
+static FILE *test_acl2_fopen(const char *path, const char *mode)
+{
+	(void)path;
+	return fopen(TEMP_ACL2_PATH, mode);
+}
+
+static FILE *test_pstat_fopen(const char *path, const char *mode)
+{
+	(void)path;
+	return fopen(TEMP_PSTAT_PATH, mode);
+}
+
+static void create_security_data_files(void)
+{
+	int fd;
+	if (0 < (fd = open(TEMP_DOXM_PATH, O_WRONLY | O_CREAT, 0644))) {
+		write(fd, TEMP_DOXM_DATA, sizeof(TEMP_DOXM_DATA));
+		close(fd);
+	}
+	if (0 < (fd = open(TEMP_CRED_PATH, O_WRONLY | O_CREAT, 0644))) {
+		write(fd, TEMP_CRED_DATA, sizeof(TEMP_CRED_DATA));
+		close(fd);
+	}
+	if (0 < (fd = open(TEMP_ACL2_PATH, O_WRONLY | O_CREAT, 0644))) {
+		write(fd, TEMP_ACL2_DATA, sizeof(TEMP_ACL2_DATA));
+		close(fd);
+	}
+	if (0 < (fd = open(TEMP_PSTAT_PATH, O_WRONLY | O_CREAT, 0644))) {
+		write(fd, TEMP_PSTAT_DATA, sizeof(TEMP_PSTAT_DATA));
+		close(fd);
+	}
+}
+
+static void remove_security_data_files(void)
+{
+	unlink(TEMP_DOXM_PATH);
+	unlink(TEMP_CRED_PATH);
+	unlink(TEMP_ACL2_PATH);
+	unlink(TEMP_PSTAT_PATH);
+}
+
+/*
+ * callbacks
+ */
+static void wm_sta_connected(wifi_manager_result_e);
```

Comment:
you'd better move these gloabal function list & struct to above of this file

Reply:
I moved all the code into the top of this file (but below global const variables).